### PR TITLE
src/im-application-list.c: Add tooltip support.

### PR DESCRIPTION
**Pull request by sunweaver**

### CODE REVIEW

**Comment by sunweaver** [comment]
@flexiondotorg It would be nice to get your feedback on this as a English native speaker. Thanks.

**Comment by trism** [comment]
Works fine, may be a bit wordy. I might have said "List of past notifications".

**Review by flexiondotorg — APPROVED** [review]
LGTM

**Review by flexiondotorg — APPROVED** [review]
LGTM

**Review by flexiondotorg — APPROVED** [review]
LGTM
